doc/news/_preparation_next_release.md

```diff
@@ -114,10 +114,30 @@ you up to date with the multi-language support provided by Elektra.
   - Updated javadoc for `KDB::close(Key)`
   - Updated javadoc for `KDB::get(Key)`
   - Updated javadoc for `KDB::get(KeySet, Key)`
-  - Updated javadoc for `KDB::set(KeySet, Key)`, better explaining the relevance of the second argument `parentKey`.
+  - Updated javadoc for `KDB::set(KeySet, Key)`, better explaining the relevance of the second argument `parentKey`
 - Introduced `KeySet::remove(Key)` and `KeySet::remove(String)`
 - Removed `KeySet::lookup(Key, int)` and `KeySet::lookup(String, int)` as well as accompanying flag definitions `KeySet::KDB_O_NONE`, `KeySet::KDB_O_DEL` and `KeySet::KDB_O_POP`. Please use `KeySet::lookup(Key)` and `KeySet::lookup(String)` instead. Instead of `KeySet::KDB_O_DEL`, please consider using `Key::release`. The proper replacement for `KeySet::KDB_O_POP` is `KeySet::remove(Key)` or `KeySet::remove(String)`.
-- Native library proxy interface `Elektra` is now package private (previously was public).
+- Native library proxy interface `Elektra` is now package private (previously was public)
+- Added example Java plugin `whitelist`
+- Added support of binary valued keys:
+  - Renamed `KeyBinaryTypeNotSupportedException` to `KeyStringValueException`
+  - Introduced `KeyBinaryValueException`
+  - Improved `Key` test coverage
+  - Introduced `Key::getBinary()` and `Key::setBinary(byte[])`
+- Fixed example project in `examples/external/java/read-keys-example`
+- Introduced abstraction `ReadOnlyKey` for better reflecting the limitations meta data keys via a type hierarchy, leading to meta data keys are now returned as `ReadOnlyKey`s:
+  - `Key` extends `ReadOnlyKey`
+  - Changed `Key Key::nextMeta()` to `Optional<ReadOnlyKey> Key::nextMeta()`, no longer throwing NoSuchElementException for non-exceptional behavior
+  - Changed `Key Key::currentMeta()` to `ReadOnlyKey Key::currentMeta()`
+  - Changed `Optional<Key> Key::getMeta(String)` to `Optional<ReadOnlyKey> Key::getMeta(String)`
+  - Meta data keys can no longer be manually released
+- `Key` class is now final
+- Removed `Key::incRef`, `Key::decRef` and `Key::getRef`
```

Comment:
```suggestion
- Removed `Key::getRef`
```

Reply:
@markus2330 I do not understand your suggestions to remove mentioning `Key::incRef` and `Key::decRef`. All three functions previously were protected and `Key` was not final. Effectively making it part of the public Java binding API.